doc/nrf/known_issues.rst

```diff
@@ -816,7 +817,7 @@ KRKNWK-8133: CSMA-CA issues
 SoftDevice Controller
 =====================
 
-.. rst-class:: v1-5-0
+.. rst-class:: v1-5-1 v1-5-0
```

Comment:
DRGN-15382 is fixed in NCS 1.5.1. Remove "(once available)" from the workaround

Reply:
Updated